### ISSUE TITLE
Did not go on training

### ISSUE THREAD

**Issue by hhhwww123** [wuhuikai/FastFCN#35]
Thanks a lot to help me solve the ‘run slave’ .However, there is a new issue happen:

/home/weizhaoxiang/anaconda3/envs/pytorch1.0/lib/python3.6/site-packages/torch/nn/_reduction.py:43: UserWarning: size_average and reduce args will be deprecated, please use reduction='mean' instead.
  warnings.warn(warning.format(ret))
Using poly LR Scheduler!
Starting Epoch: 0
Total Epoches: 240
  0%|                                                                                                                                                                | 0/371 [00:00<?, ?it/s]
=>Epoches 0, learning rate = 0.0050,                 previous best = 0.0000
/home/weizhaoxiang/anaconda3/envs/pytorch1.0/lib/python3.6/site-packages/torch/nn/functional.py:2390: UserWarning: nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.functional.upsample is deprecated. Use nn.functional.interpolate instead.")
problem. 

After the print, the code did not go on training or break, just stop at here. And you can not check there is a thread exist with the command  'nvidia-smi'.

How to solve the problem??

**Comment by wuhuikai** [comment]
1. Try to run it again and wait for a few minutes
2. `Ctrl + C` and then report the error message

**Comment by hhhwww123** [comment]
Thanks，it work！！But it's thread still can't be found exist with the command 'nvidia-smi'

**Comment by wuhuikai** [comment]
Can not figure out it :(
It works well on my machine.

**Comment by wuhuikai** [comment]
@hhhwww123 The code in branch `latest` is now ready, which uses official `SyncBatchNorm` and can run on all OS with `PyTorch>=1.1.0`.
